### PR TITLE
feat(conference): add Theme.layout

### PR DESCRIPTION
**Pull request by drymarau**
This should provide read-only access to currently active layout of the conference.

Ability to transform the layout will come at a later stage.

### CODE REVIEW

File: sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Screen.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+public data class Screen(
+    @SerialName("chair_layout")
+    val hostLayout: LayoutId,
+    @SerialName("guest_layout")
+    val guestLayout: LayoutId,
```
Comment (tpettersen) [inline sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Screen.kt:26]: According to the docs it is supposed to be "chair" and "guest" 🤔 Based on the response you use in the tests, I guess the docs are misleading

File: sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Event.kt [hunk 2]
```diff
@@ -80,6 +80,16 @@ public data class MessageReceivedEvent(
     val direct: Boolean = false,
 ) : Event
 
+@Serializable
+public data class LayoutEvent(
```
Comment (tpettersen) [inline sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Event.kt:84]: No need for participants yet?

**Review by tpettersen — APPROVED** [review]
Looks good :)

File: sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Event.kt (hunk 2 above)
Comment (drymarau) [reply sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Event.kt:84]: No clear idea how to use that property in an Android client to be honest. Easy to add later with an empty `Set` as a default value without breaking the clients.

File: sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Screen.kt (hunk 1 above)
Comment (drymarau) [reply sdk-api/src/main/kotlin/com/pexip/sdk/api/infinity/Screen.kt:26]: Test file is from Infinity, while I'm not sure where the one in docs comes from.

**Pull request merged**